ledger/participant-integration-api/src/main/resources/db/migration/h2database/V1__Append_only_schema.sql

```diff
@@ -399,12 +399,13 @@ CREATE TABLE participant_identity_provider_config
 -- Participant local store: users
 ---------------------------------------------------------------------------------------------------
 CREATE TABLE participant_users (
-    internal_id         INTEGER             GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    user_id             VARCHAR(256)        NOT NULL UNIQUE,
-    primary_party       VARCHAR(512),
-    is_deactivated      BOOLEAN             NOT NULL,
-    resource_version    BIGINT              NOT NULL,
-    created_at          BIGINT              NOT NULL
+    internal_id          INTEGER GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```

Comment:
Is it on purpose that you're removing a nice columnar alignment from before? (here and participant_party_records table)

Reply:
I tried to recover it now :)